pages/orders/[orderId]/index.tsx

```diff
@@ -1,129 +1,85 @@
-import { Badge, Flex, Grid, GridItem, H3, Panel } from '@bigcommerce/big-design';
+import { Button, Flex, H1, Panel, StatefulTable, Text } from '@bigcommerce/big-design';
 import { useRouter } from 'next/router';
 import React from 'react';
 import styled from 'styled-components';
 
 import ErrorMessage from '../../../components/error';
 import Loading from '../../../components/loading';
 import { useOrder } from '../../../lib/hooks';
-import { Order } from '../../../types';
+import { BillingAddress, OrderProduct, ShippingOrder } from '../../../types';
+
+interface InternalOrderPageProps {
+    order: ShippingOrder;
+}
 
 const StyledAddress = styled.address`
-  color: ${({ theme }) => theme.colors.secondary70};
-  font-style: normal;
-  line-height: ${({ theme }) => theme.lineHeight.medium};
+    color: ${({ theme }) => theme.colors.secondary70};
+    font-style: normal;
+    line-height: ${({ theme }) => theme.lineHeight.small};
 `;
 
-const StyledDl = styled.dl`
-  color: ${({ theme }) => theme.colors.secondary70};
-  display: grid;
-  grid-template: 'dt dd' auto / 1fr auto;
-  grid-auto-rows: auto;
-  line-height: ${({ theme }) => theme.lineHeight.medium};
-  margin: 0;
-
-  dt {
-    grid-area: 'dt';
-  }
+const InternalOrderPage = (props: InternalOrderPageProps) => {
+    const { shipping_addresses = [], products = [] } = props.order
+    const items = shipping_addresses && shipping_addresses.map(address => {
+        const addressProducts = products && products.filter(product => product.order_address_id === address.id);
 
-  dd {
-    grid-area: 'dd';
-    text-align: right;
-  }
-`;
+        return { address, addressProducts }
+    })
 
-const StyledSpan = styled.span`
-  font-weight: ${({ theme }) => theme.typography.fontWeight.semiBold};
-`;
+    const renderAddress = (address: BillingAddress) => (
+        <StyledAddress>
+            <div>{address.first_name} {address.last_name}</div>
+            <div>{address.street_1} {address.street_2}</div>
+            <div>{address.city}, {address.state} {address.zip}</div>
+        </StyledAddress>
+    )
 
-const InternalOrderPage: React.FC<{ order: Order }> = ({ order }) => {
-  const { billing_address } = order;
-  
+    const renderProducts = (products: OrderProduct[]) => (
+        <>
+            {products.map(product => <Text key={product.name}>{product.name}</Text>)}
+        </>
+    )
```

Comment:
The design actually has margins between products. I think it makes sense to distinguish them as a list of items rather than a unit like the addresses.

Reply:
Sounds good, might want to use a `ul` + `li` then. We have a helper on the theme for that.
```tsx
const StyledUl = styled.ul`
    ${({ theme }) => theme.helpers.listReset}

    color: ${({ theme }) => theme.colors.secondary70};
`
```